### PR TITLE
nose2pytest migration batch 1

### PR DESCRIPTION
**Pull request by kloczek**
Most of the changes done by sed -i s/nt.assert/assert/ -> use
nose2pytest script from `nose2pytest` module from
https://github.com/schollii/nose2pytest/
At the end some manual changes (mostly about proper pytest module
import).

Signed-off-by: Tomasz Kłoczko <kloczek@github.com>

### CODE REVIEW

File: IPython/core/tests/test_logger.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def test_logstart_inaccessible_file():
     except IOError:
         pass
     else:
-        nt.assert_true(False)           # The try block should never pass.
+        assert False                    # The try block should never pass.
```
Comment (MrMino) [inline IPython/core/tests/test_logger.py:15]:
```suggestion
        pytest.fail("The try block should never pass.")
```

**Comment by MrMino** [comment]
LGTM, just let me darken this...

**Review by MrMino — APPROVED** [review]

**Comment by kloczek** [comment]
Thx

**Comment by MrMino** [comment]
@kloczek sorry it took so long, I forgot to merge. Thanks for the PR!

**Pull request merged**